src/components/SEO/helper.ts

```diff
@@ -70,7 +70,7 @@ export const buildMetadata = (
     })
   }
   if (image) {
-    const imageUrl = siteUrl + getSrc(image)
+    const imageUrl = siteUrl + (getSrc(image) || image)
```

Comment:
Q: why do we need this `||`?

Reply:
Our stable urls are only created in production. So I configured the code to send `SEO` a `gatsby-plugin-image` object if the site is running in development and send `SEO` the stable url string if the site is running in production. So `imageUrl` will either be the default src of a `gatsby-plugin-image` or just the image itself.

But if we don't care if the meta tags in development are showing a image that doesn't exist, we can just take this off and put `const imageUrl = siteUrl + image`.